src/PostgREST/Main.hs

```diff
@@ -13,22 +12,21 @@ import           PostgREST.Middleware
 import           Control.Monad                        (unless)
 import           Control.Monad.IO.Class               (liftIO)
 import           Data.Functor.Identity
-import           Data.List                            (intercalate)
-import           Data.String.Conversions              (cs)
+import           Data.String.Conversions              (cs,
+                                                       (<>))
```

Comment:
The rest of the code typically imports `<>` from Data.Monoid, would that work here as well?